doc/sphinx-guides/source/developers/intro.rst

```diff
@@ -29,7 +29,7 @@ We make use of a variety of Java EE technologies such as JPA, JAX-RS, JMS, and J
 Issue Tracker
 -------------
 
-We use GitHub Issues as our issue tracker: https://github.com/IQSS/dataverse/issues
+We use GitHub Issues as our issue tracker: https://github.com/CeON/dataverse/projects/1
```

Comment:
This link links to our sprint board – is it ok?

Reply:
Yeah it's fine, in the next pr I will remove kanban board sections and merge them into one.